ui/src/dashboard/menus/MenuForm.tsx

```diff
@@ -51,6 +52,7 @@ export const MenuForm = ({ onSubmit, menu = null }) => {
                 <Field isFormik name="title" label="Title" />
                 <Field isFormik as="textarea" label="Description" name="description" />
                 <Field isFormik as="textarea" label="Disclaimer" name="disclaimer" />
+                <Multi />
```

Comment:
We probably want to pass the value for notes coming from Formik here so that what we see withiin Multi isnt just UI and we can actually pass it to formik's initial values. Let me know if you need an example